src/main/scala/Keys.scala

```diff
@@ -1,6 +1,6 @@
 package kotlin
 
-import sbt._
+import sbt.*
```

Comment:
sbt metabuild now has `-Xsource:3` enabled, and we get the corresponding warnings.